Week1/homework/queries.js

```diff
@@ -0,0 +1,114 @@
+var mysql      = require('mysql');
+var connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  database : 'myworld'
+});
+
+connection.connect(function(err) {
+    if (err) throw err;
+    console.log('db connected');
+  });
+
+// Question 4i: What are the names of the countries with population greater than 8 million
+let result4i = 'SELECT Name FROM Country WHERE Population > 8000000;'
+
```

Comment:
I know you have just copy past but I still don't like the ';' , usually we always avoid unnecessary character.